eZ/Bundle/EzPublishCoreBundle/Features/QueryController/query_controller.feature

```diff
@@ -2,7 +2,8 @@ Feature: Query controller
     In order to simplify listing items from the repository
     As a developer
     I want to run repository queries from content views
-
+    
+@admin
```

Comment:
why? QueryController is not an admin feature strictly

Reply:
This is a BehatBundle feature. It used to work like this:
any Context using RepositoryContext trait would automatically use the repository as Admin.(https://github.com/ezsystems/ezpublish-kernel/pull/2642/files#diff-2ea4ba4849b978d5417e2c5f8470cf87L49)

This is now changed, the default user is Anonymous - it can be set manually using a step like "I am using the API as :username" or a scenario can be tagged as `@admin` to have Admin user set automatically. (https://github.com/ezsystems/BehatBundle/blob/master/src/lib/API/Context/TestContext.php#L34) I believe this allows us to have more control over the tests, as not all tests should execute using admin rights.

I agree that the tag might be a bit misleading without this context.